From eb876030f29b53cbb01f4fc1fc61b0c0159f10d8 Mon Sep 17 00:00:00 2001
From: Jerry Cao <jerry.cao@amlogic.com>
Date: Thu, 9 Oct 2014 15:25:42 -0700
Subject: [PATCH 4933/5965] [PD #97597] [OSD] Yet another compilation fix for
 Meson6 family SoC

The RDMA register is on CBUS instead of VCBUS for chipset prior to Meson8.

Change-Id: I17483bc3a41fe43dabd351cef00a9ce01dd73e79
---
 drivers/amlogic/display/osd/osd_rdma.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/amlogic/display/osd/osd_rdma.c b/drivers/amlogic/display/osd/osd_rdma.c
index 9b542a5a7d82..4134673c106c 100755
--- a/drivers/amlogic/display/osd/osd_rdma.c
+++ b/drivers/amlogic/display/osd/osd_rdma.c
@@ -217,7 +217,7 @@ int reset_rdma(void)
 						aml_read_reg32(REGS_RECONFIG(rdma_table[check_number].addr)))
 				{
 					printk("the rdma write error addr is 0x%x, the old value is 0x%x, the real value is 0x%x\n",
-							rdma_table[check_number].addr, aml_read_reg32(VCBUS_REG_ADDR(rdma_table[check_number].addr)),
+							rdma_table[check_number].addr, aml_read_reg32(REGS_RECONFIG(rdma_table[check_number].addr)),
 							rdma_table[check_number].val);
 					check_rdma_table[has_checked].addr = rdma_table[check_number].addr;
 					has_checked++;
-- 
2.19.0

